src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1114,6 +1115,14 @@ private function registerWorkflowConfiguration(array $config, ContainerBuilder $
             // Global transition counter per workflow
             $transitionCounter = 0;
             foreach ($workflow['transitions'] as $transition) {
+                foreach (['from', 'to'] as $direction) {
+                    foreach ($transition[$direction] as $k => $arc) {
+                        $transition[$direction][$k] = new Definition(Arc::class, [
+                            '$place' => $arc['place'],
+                            '$weight' => $arc['weight'] ?? 1,
+                        ]);
```

Comment:
let's save on reflection:
```suggestion
                        $transition[$direction][$k] = new Definition(Arc::class, [$arc['place'], $arc['weight'] ?? 1]);
```